sirepo/package_data/static/js/raydata.js

```diff
@@ -637,25 +692,48 @@ SIREPO.app.directive('scanSelector', function() {
 		<div class="clearfix"></div>
                 <button type="submit" class="btn btn-primary" data-ng-show="showSearchButton()" data-ng-click="search()">Search</button>
 	      </form>
+	          <button class="btn btn-info btn-xs" data-ng-click="newCommand()" style="float: right;"><span class="glyphicon glyphicon-plus"></span></button>
               <table class="table table-striped table-hover col-sm-4">
                 <thead>
                   <tr>
-                    <th data-ng-repeat="h in getHeader()">{{ h }}</th>
-                  </tr>
+                    <th data-ng-repeat="column in columns track by $index" data-ng-mouseover="hoverChange($index, true)" data-ng-mouseleave="hoverChange($index, false)" style="width: 100px; height: 40px;">
+                          {{ column }}
+                          <button type="submit" class="btn btn-primary btn-xs"  id="del-column" data-ng-show="showDeleteButton($index)" data-ng-click="deleteCol(column)"><span class="glyphicon glyphicon-remove"></span></button>
+                    </th>
+                </tr>
                 </thead>
                 <tbody ng-repeat="s in scans">
                   <tr>
-                    <td><input type="checkbox" data-ng-checked="s.selected" data-ng-click="toggleScanSelection(s)"/></td>
-                    <td data-ng-repeat="c in getHeader().slice(1)">{{ getScanField(s, c) }}</td>
+                    <td style="width: 100px; height: 40px;"><input type="checkbox" data-ng-checked="s.selected" data-ng-click="toggleScanSelection(s)"/></td>
+                    <td data-ng-repeat="c in columns.slice(1)" style="width: 100px; height: 40px;">{{ getScanField(s, c) }}</td>
                   </tr>
                 </tbody>
               </table>
 	    </div>
+
+	    <div data-element-picker="" data-controller="control" data-title="New Command" data-id="sr-newCommand-editor" data-small-element-class="col-sm-3" data-available-columns="availableColumns" data-master-list-columns="masterListColumns" data-columns="columns"></div>
```

Comment:
Title could be changed to something like "Add Column".

Reply:
id could be changed to something like `sr-columnPicker`

Reply:
07e13b7b7da132d5450c7714b0bcd7a04982308c